# HG changeset patch
# User Rik <rik@octave.org>
# Date 1472671478 25200
#      Wed Aug 31 12:24:38 2016 -0700
# Node ID 2bc07741efa0db564ba402c1e06b70608e78d025
# Parent  c69805d1fa643e94e4de25c43f8e9f78f614c7be
maint: Style check C++ code in src/ directory.

* display-available.h, main-cli.cc, main-gui.cc, main.in.cc, mkoctfile.in.cc,
octave-config.in.cc, shared-fcns.h:
Style check C++ code in src/ directory.

diff --git a/src/display-available.h b/src/display-available.h
--- a/src/display-available.h
+++ b/src/display-available.h
@@ -30,8 +30,9 @@ extern "C" {
 
 extern const char *display_available (int *dpy_avail);
 
 #if defined (__cplusplus)
 }
 #endif
 
 #endif
+
diff --git a/src/main-cli.cc b/src/main-cli.cc
--- a/src/main-cli.cc
+++ b/src/main-cli.cc
@@ -84,8 +84,9 @@ main (int argc, char **argv)
   check_hg_versions ();
 
   octave::sys::env::set_program_name (argv[0]);
 
   octave::cli_application app (argc, argv);
 
   return app.execute ();
 }
+
diff --git a/src/main-gui.cc b/src/main-gui.cc
--- a/src/main-gui.cc
+++ b/src/main-gui.cc
@@ -98,8 +98,9 @@ main (int argc, char **argv)
   check_hg_versions ();
 
   octave::sys::env::set_program_name (argv[0]);
 
   octave::gui_application app (argc, argv);
 
   return app.execute ();
 }
+
diff --git a/src/main.in.cc b/src/main.in.cc
--- a/src/main.in.cc
+++ b/src/main.in.cc
@@ -414,8 +414,9 @@ batch-oriented language for data process
 GNU Octave is free software. You may redistribute it and/or modify it
 under the terms of the <a href="http://www.gnu.org/licenses/">GNU
 General Public License</a> as published by the Free Software Foundation.
 
 This is the developer documentation for Octave's own source code. It is
 intended to help for hacking Octave. It may also be useful for
 understanding the Octave API when writing your own .oct files.
 */
+
diff --git a/src/mkoctfile.in.cc b/src/mkoctfile.in.cc
--- a/src/mkoctfile.in.cc
+++ b/src/mkoctfile.in.cc
@@ -623,18 +623,18 @@ main (int argc, char **argv)
           while (! feof (fd))
             {
               line = get_line (fd);
               if ((pos = line.rfind (".o:")) != std::string::npos)
                 {
                   size_t spos = line.rfind ('/', pos);
                   std::string ofile =
                     (spos == std::string::npos
-                      ? line.substr (0, pos+2)
-                      : line.substr (spos+1, pos-spos+1));
+                     ? line.substr (0, pos+2)
+                     : line.substr (spos+1, pos-spos+1));
                   fo << "pic/" << ofile << " " << ofile << " "
                      << dfile << line.substr (pos) << std::endl;
                 }
               else
                 fo << line << std::endl;
             }
           pclose (fd);
           fo.close ();
@@ -657,18 +657,18 @@ main (int argc, char **argv)
           while (! feof (fd))
             {
               line = get_line (fd);
               if ((pos = line.rfind (".o:")) != std::string::npos)
                 {
                   size_t spos = line.rfind ('/', pos);
                   std::string ofile =
                     (spos == std::string::npos
-                      ? line.substr (0, pos+2)
-                      : line.substr (spos+1, pos-spos+1));
+                     ? line.substr (0, pos+2)
+                     : line.substr (spos+1, pos-spos+1));
                   fo << "pic/" << ofile << " " << ofile << " "
                      << dfile << line.substr (pos+2) << std::endl;
                 }
               else
                 fo << line << std::endl;
             }
           pclose (fd);
           fo.close ();
@@ -819,8 +819,9 @@ main (int argc, char **argv)
         {
           std::string cmd = "strip " + octfile;
           result = run_command (cmd);
         }
     }
 
   return result;
 }
+
diff --git a/src/octave-config.in.cc b/src/octave-config.in.cc
--- a/src/octave-config.in.cc
+++ b/src/octave-config.in.cc
@@ -169,8 +169,9 @@ main (int argc, char **argv)
           std::cerr << "octave-config: unrecognized argument " << arg
                     << std::endl;
           return 1;
         }
     }
 
   return 0;
 }
+
diff --git a/src/shared-fcns.h b/src/shared-fcns.h
--- a/src/shared-fcns.h
+++ b/src/shared-fcns.h
@@ -140,8 +140,9 @@ subst_octave_home (const std::string& s)
 
   if (dir_sep_char != '/')
     std::replace (retval.begin (), retval.end (), '/', dir_sep_char);
 
   return retval;
 }
 
 #endif
+
